packages/language-server/test/utils.ts

```diff
@@ -12,11 +12,11 @@ import { pathToUrl } from '../src/utils';
  * @returns
  */
 export function createEnvironment(filePath: string, baseDir: string, pathPrefix?: string) {
-	const fixtureDir = join(__dirname, 'plugins', baseDir, 'fixtures', pathPrefix ?? '');
+	const fixtureDir = join(__dirname, 'plugins', baseDir, 'fixtures');
```

Comment:
Small change to make tests faster, previously we'd set the root of the workspace in the specific folder of the fixtures (so for instance, `fixtures/diagnostics`) instead of the general folder where fixtures are (`fixtures`).

Since there's no `tsconfig.json` in those folders, the language service would end up going up the directories and loading a bunch of unrelated files which slow down things massively